projects/schematics/src/dependencies.json

```diff
@@ -55,8 +55,10 @@
     "@angular/common": "^13.3.0",
     "@angular/core": "^13.3.0",
     "@angular/forms": "^13.3.0",
+    "@ng-select/ng-select": "^8.0.0",
     "@ngrx/effects": "^13.0.2",
     "@ngrx/store": "^13.0.2",
+    "@spartacus/cart": "4.1.0-next.0",
```

Comment:
Let's discuss to see if there is an alternative to adding a dependency to the cart in ASM